gap/semigroups/semieunit.gi

```diff
@@ -0,0 +1,715 @@
+#############################################################################
+##
+#W  semieunit.gi
+#Y  Copyright (C) 2017                                    Christopher Russell
+##
+##  Licensing information can be found in the README file of this package.
+##
+#############################################################################
+#
+# This code is based on the description of McAlister triple semigroups found
+# in Fundamentals of Semigroup Theory by John Howie.
+#
+#############################################################################
+# Methods for creating McAlister triple semigroups
+#############################################################################
+InstallMethod(McAlisterTripleSemigroup,
+"for a perm group, action, digraph, and digraph",
+[IsGroup, IsFunction, IsDigraph, IsDigraph],
```

Comment:
I think that line checks whether the `Image(hom)` is a subgroup (Image(hom) is the representation of the action of the group as a perm group) so this should be fine!